.cirrus.yml

```diff
@@ -41,36 +41,77 @@ coverage_environment_template: &COVERAGE_ENVIRONMENT_TEMPLATE
   CODECOV_TOKEN: ENCRYPTED[cc6ce01618eaee6c9a08ebc81446a9233588df3b2c6ba4e30ffb7715ee9815734ccfd9fcec4f6abfc6a1b9dd8253110d]
 
 # Linux
+linux_task_matrix_template: &LINUX_TASK_MATRIX_TEMPLATE
+  - TASK_NAME_SUFFIX: x86, DMD (latest)
+    MODEL: 32
+  - TASK_NAME_SUFFIX: x86, DMD (coverage)
+    MODEL: 32
+    << : *COVERAGE_ENVIRONMENT_TEMPLATE
+  - TASK_NAME_SUFFIX: x86, DMD (bootstrap)
+    MODEL: 32
+    HOST_DMD: dmd-2.079.0
+  - TASK_NAME_SUFFIX: x64, DMD (latest)
+  # Enable this to replace coverage tests on CircleCI
+  # - TASK_NAME_SUFFIX: x64, DMD (coverage)
+  #   << : *COVERAGE_ENVIRONMENT_TEMPLATE
+  - TASK_NAME_SUFFIX: x64, DMD (bootstrap)
+    HOST_DMD: dmd-2.079.0
+  - TASK_NAME_SUFFIX: x64, LDC
+    HOST_DC: ldc #TODO: Update to HOST_DMD when support for HOST_DC is removed
+  - TASK_NAME_SUFFIX: x64, GDC
+    HOST_DMD: gdmd-9
+
 task:
-  name: Ubuntu 16.04 $TASK_NAME_SUFFIX
+  name: Ubuntu 20.04 $TASK_NAME_SUFFIX
   container:
-    image: ubuntu:16.04
+    image: ubuntu:20.04
```

Comment:
This version 20.04 is effectively discontinued now, too. (Technically, it’s still available under the ESM thing, but I don’t think that counts.)